client.go

```diff
@@ -1044,6 +1123,89 @@ func (c *InternalHTTPClient) RowAttrDiff(ctx context.Context, index, frame strin
 	return rsp.Attrs, nil
 }
 
+func (c *InternalHTTPClient) getContents(ctx context.Context, u url.URL) ([]byte, error) {
+	// Build request.
+	req, err := http.NewRequest("GET", u.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+	// Execute request.
+	resp, err := c.HTTPClient.Do(req.WithContext(ctx))
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	// Return error if status is not OK.
+	switch resp.StatusCode {
+	case http.StatusOK: // ok
+	case http.StatusNotFound:
+		return nil, ErrIndexNotFound
+	default:
+		return nil, fmt.Errorf("unexpected status: code=%d", resp.StatusCode)
+	}
+
+	var buf bytes.Buffer
+	w := bufio.NewWriter(&buf)
+	if _, err = io.Copy(w, resp.Body); err != nil {
+		return nil, err
+	}
+	w.Flush()
+	return buf.Bytes(), nil
+
+}
+
+// FetchRawColumnAttrs retrieves the raw bolt data file contents of the Column Attributes
+func (c *InternalHTTPClient) FetchRawColumnAttrs(ctx context.Context, indexName string) ([]byte, error) {
```

Comment:
it doesn't appear that either of these `FetchRaw*` methods (or `getContents()`) are being used anywhere.